app/components/settings/categories/DisplaySettings.js

```diff
@@ -4,9 +4,9 @@ import classnames from 'classnames';
 import { observer } from 'mobx-react';
 import { defineMessages, intlShape } from 'react-intl';
 import styles from './DisplaySettings.scss';
-import themeDefault from '../../../assets/images/themes/theme-default.png';
-import theme1 from '../../../assets/images/themes/theme-1.png';
-import theme2 from '../../../assets/images/themes/theme-2.png';
+import themeLightBlue from '../../../assets/images/themes/light-blue.png';
+import themeCardano from '../../../assets/images/themes/cardano.png';
+import themeDarkBlue from '../../../assets/images/themes/dark-blue.png';
```

Comment:
I think adding `Preview` at the end of the imported images would increase readability here. e.g: `themeLightBluePreview`

Reply:
@DominikGuzei agreed and fixed ;)